src/libponyrt/actor/actor.c

```diff
@@ -293,9 +293,12 @@ static void try_gc(pony_ctx_t* ctx, pony_actor_t* actor)
 
 static void send_unblock(pony_actor_t* actor)
```

Comment:
might be worth renaming this to `unblock` given the changes to the logic..

Reply:
lol i did that then changed it back.